stories/src/ui/Tabs.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
+import { Tabs, Tab } from '@datapunt/asc-ui'
+
+<Meta title="UI/Tabs" component={Tabs} />
+
+# Tabs
+
+Tabs organise and allow navigation between groups of content that are related and at the same level of hierarchy.
+
+**Component status: experimental**
+
+## Default
+
+<Preview>
+  <Story name="Default">
+    <Tabs label="An example of tabs">
+      <Tab id="one" label="First">
+        Contents of the first tab.
+      </Tab>
+      <Tab id="two" label="Second">
+        Contents of the second tab.
+      </Tab>
+      <Tab id="three" label="Last">
+        Contents of the last tab.
+      </Tab>
+    </Tabs>
+  </Story>
+</Preview>
```

Comment:
I saw that it also works well on mobile with the horizontal scroll, nice! Should we add a mobile story here as well just for convenience? You can use a storybook addon for this